software/config/drc_environment.sh

```diff
@@ -114,6 +114,27 @@ setup_robot_computers()
   fi
 }
 
+setup_valkyrie_computers()
+{
+  # Edinburgh operator workstations
+  if [ "angmar" = $(hostname) ] || [ "gondolin" = $(hostname) ] || [ "vis04" = $(hostname) ]; then
```

Comment:
From what I've seen, Kody must have renamed the operator workstation to vis04 since this is what shows up on the terminal ($edbot@vis04). I'd take one of the other two of the list but am unsure which one is the video capture station and which one the workstation computer.